ranger/container/settings.py

```diff
@@ -107,6 +108,8 @@
     'cd_tab_case': ['sensitive', 'insensitive', 'smart'],
     'confirm_on_delete': ['multiple', 'always', 'never'],
     'draw_borders': ['none', 'both', 'outline', 'separators'],
+    'draw_borders_multipane': ['none', 'both', 'outline',
```

Comment:
```suggestion
    'draw_borders_multipane': [None, 'none', 'both', 'outline',
```
Like this?

Reply:
First off, my comment did assume `None` was a valid setting. But I didn't mean you should copy my words verbatim. I'm not going to say "`None` or the equivalent value" all the time, just need to get my point across.

I'm not sure we should use `None`, settings should be case-insensitive and I'm not 100% on how `set draw_borders none` would be handled if we assume only `None` means no borders. If it works fine then I'm ok with it. But do not forget that "false" is an expected value too for compat reasons.

Reply:
Distinguishing between no setting at all and being explicitly set to "none" does complicate matters a bit.
Let's also drop the cleverness of `or` and make it explicit with `if setting is None` or something, truthiness is such an ugly implicit thing in python.

Reply:
Apologies if that came off that way. I meant that as in that was totally 100% my fault for misunderstanding you.

`None` would only be the set value if the user never explicitly specifies the setting for `set draw_borders_multipane` and to instead fallback onto `set draw_borders`, it would be impossible to manually set `None`, only `'none'` for no borders.

I'll check through the logic and make sure `"false"` is properly handled too. Hopefully ASAP.

Reply:
No offense was experienced. Just wanted to clarify.
No rush either, take your time.